ui/app/models/auth-config/github.js

```diff
@@ -1,24 +1,28 @@
 import { computed } from '@ember/object';
 import DS from 'ember-data';
-
 import AuthConfig from '../auth-config';
 import fieldToAttrs from 'vault/utils/field-to-attrs';
+import { combineFieldGroups } from 'vault/utils/openapi-to-attrs';
 
 const { attr } = DS;
 
 export default AuthConfig.extend({
+  useOpenAPI: true,
```

Comment:
whether or not this model class should be re-opened and extended with additional fields from OpenAPI.